Configuration/PyReleaseValidation/python/relval_2023.py

```diff
@@ -22,7 +22,7 @@
 numWFIB.extend([22034.0]) #2023D23
 numWFIB.extend([22434.0]) #2023D24
 numWFIB.extend([24034.0]) #2023D28
-numWFIB.extend([27434.0,27434.21,27634.21]) #2023D35, prodlike, prodlike PU
+numWFIB.extend([27434.0,27434.21]) #2023D35, prodlike
```

Comment:
@kpedro88 adding the missing input to fix the IB problem is ok, but why removing the test? Didi you try its timing? was it running in TimeOut?

Reply:
I didn't time it, but knowing how long it takes to run 200PU, it will definitely time out.

Reply:
Well, I did a test on cmsdev15, using 4 threads in parallel for 27634.21, it took about 40 minutes to run:

09:39 cmsdev15 1284> ls -lrt | grep log | awk '{print $8,$9}'
19:20 step1_TTbar_14TeV_ProdLike+TTbar_14TeV_TuneCUETP8M1_2023D35PU_GenSimHLBeamSpotFull14+DigiFullTriggerPU_2023D35PU+RecoFullGlobalPU_ProdLike_2023D35PU+MiniAODFullGlobalPU_ProdLike_2023D35PU.log
19:38 step2_TTbar_14TeV_ProdLike+TTbar_14TeV_TuneCUETP8M1_2023D35PU_GenSimHLBeamSpotFull14+DigiFullTriggerPU_2023D35PU+RecoFullGlobalPU_ProdLike_2023D35PU+MiniAODFullGlobalPU_ProdLike_2023D35PU.log
19:52 step3_TTbar_14TeV_ProdLike+TTbar_14TeV_TuneCUETP8M1_2023D35PU_GenSimHLBeamSpotFull14+DigiFullTriggerPU_2023D35PU+RecoFullGlobalPU_ProdLike_2023D35PU+MiniAODFullGlobalPU_ProdLike_2023D35PU.log
19:55 step4_TTbar_14TeV_ProdLike+TTbar_14TeV_TuneCUETP8M1_2023D35PU_GenSimHLBeamSpotFull14+DigiFullTriggerPU_2023D35PU+RecoFullGlobalPU_ProdLike_2023D35PU+MiniAODFullGlobalPU_ProdLike_2023D35PU.log

This would be likely the longest test in IB, but below the 2 hours TimeOut limits we have (@smuzaffar please confirm). Of course it depends on the machine performances. As PR tests run in single threaded mode though, if the original idea was to use it for PR tests it would not work (assuming naively a factor of 4 longer run time).

Reply:
Yes for IBs, we have per step timeout limit of 2h.

Reply:
what could be an issue is the memory consumprion. For step2 the SimpleMemoryChecker tells me:

MemoryReport> Peak virtual size 12871.8 Mbytes
 Key events increasing vsize: 
[0] run: 0 lumi: 0 event: 0  vsize = 0 deltaVsize = 0 rss = 0 delta = 0
[1] run: 1 lumi: 1 event: 2  vsize = 10557 deltaVsize = 0 rss = 8058.27 delta = 0
[2] run: 1 lumi: 1 event: 4  vsize = 11325 deltaVsize = 768 rss = 7353.9 delta = -704.363
[4] run: 1 lumi: 1 event: 1  vsize = 11847 deltaVsize = 522 rss = 6769.11 delta = -584.789
[5] run: 1 lumi: 1 event: 5  vsize = 12359.7 deltaVsize = 512.762 rss = 8011.16 delta = 1242.05
[8] run: 1 lumi: 1 event: 10  vsize = 12359.8 deltaVsize = 0.0703125 rss = 6721.27 delta = -335.773
[10] run: 1 lumi: 1 event: 6  vsize = 12871.8 deltaVsize = 0 rss = 5853.9 delta = -574.703
[9] run: 1 lumi: 1 event: 9  vsize = 12871.8 deltaVsize = 512 rss = 6428.6 delta = -292.668

and for step3

MemoryReport> Peak virtual size 14971.2 Mbytes
 Key events increasing vsize: 
[2] run: 1 lumi: 1 event: 3  vsize = 12786.2 deltaVsize = 512.363 rss = 8542.52 delta = -23.8242
[3] run: 1 lumi: 1 event: 4  vsize = 13554.2 deltaVsize = 768 rss = 8645.19 delta = 102.676
[5] run: 1 lumi: 1 event: 5  vsize = 14970.2 deltaVsize = 1416 rss = 8388.85 delta = -256.34
[6] run: 1 lumi: 1 event: 7  vsize = 14971.2 deltaVsize = 0.929688 rss = 8555.35 delta = 166.5
[0] run: 0 lumi: 0 event: 0  vsize = 0 deltaVsize = 0 rss = 0 delta = 0
[8] run: 1 lumi: 1 event: 10  vsize = 14971.2 deltaVsize = 0 rss = 7559.48 delta = -995.871
[7] run: 1 lumi: 1 event: 8  vsize = 14971.2 deltaVsize = 0 rss = 8570.72 delta = 15.3672
[6] run: 1 lumi: 1 event: 7  vsize = 14971.2 deltaVsize = 0.929688 rss = 8555.35 delta = 166.5